src/protocols/aeternity/views/TransactionDetails.vue

```diff
@@ -309,44 +303,40 @@ export default defineComponent({
     });
 
     onMounted(async () => {
-      let rawTransaction = getTransactionByHash(activeAccount.value.address, hash);
+      let rawTransaction: ITransaction | undefined;
 
-      if (!rawTransaction || rawTransaction.incomplete) {
+      try {
```

Comment:
So with current logic the user would need to fetch transaction every time he would enter the `TransactionDetails` page. Do the benefits of the new approach outweigh the reduction in user convenience?

Reply:
Good finding. At some point as there was no `transactionList` composable anymore we needed to fetch the transaction every time we open the details page. But I brought it and now we are able to pick it from cache if we are navigation within one list of transactions. Please see my change.